crates/matrix-sdk-crypto/src/identities/user.rs

```diff
@@ -622,6 +619,19 @@ impl ReadOnlyOwnUserIdentity {
 
         Ok(())
     }
+
+    fn filter_devices_to_request(
+        &self,
+        devices: HashMap<OwnedDeviceId, ReadOnlyDevice>,
```

Comment:
Is it mandatory to takes `devices` by value here? Could a reference work?

Reply:
We need to return a `Vec<OwnedDeviceId>` so this avoids to clone the IDs, especially given that the list is unused after.

Reply:
Good to me then.